### PR TITLE
Chore: PR feedback on chappie2.0

### PR DESCRIPTION
**Pull request by robinpiets**
Fixed feedback on #414:
- Removed unnecessary layout defintion ad28042
- Updated comment 9a68632
- Removed otherProps and ordered props by name c2eb858
- Refactored useEffect in two separate useEffects a625dac
- Moved the return statement higher up the tree 23f4520
- Wrapped all if statements in brackets 2a1654d
- Refactored usePassPropsToChildren and renamed childrenProps 09f243d
- Removed addressPage 83abfcb
- Removed unnecessary or statement 9ef3b68
- Switched routes and reordered props 70f5037
- Disabled the `calculation of the bundle size ` b71e317
- Removed unnecessary comments b711dc0
- Renamed usePassPropsToChildren to passPropsToChildren bdd5346

### CODE REVIEW

File: apps/client/src/components/StepByStepNavigation/StepByStepNavigation.tsx [hunk 1]
```diff
@@ -8,21 +8,25 @@ import StepByStepNavigationStyle, { Props } from "./StepByStepNavigationStyle";
...
+  // Pass the props defined below to all the children
+  const { children } = usePassPropsToChildren(childrenProp, {
     customSize,
     disabledTextColor,
     doneTextColor,
     highlightActive,
   });
 
-  if (!childrenProps) return null;
+  // React Hook "usePassPropsToChildren" must be called before this `return` statement
```
Comment (robinpiets) [inline apps/client/src/components/StepByStepNavigation/StepByStepNavigation.tsx]: Or we can use useEffect() 🙄 

Edit: doesn't work..

File: apps/client/src/components/Layouts/BaseLayout.tsx [hunk 2]
```diff
@@ -33,9 +32,9 @@ function BaseLayout({ children, checker, heading }: BaseLayoutProps) {
             span={{
               small: 1,
               medium: 2,
-              big: checker ? 6 : 5,
-              large: checker ? 12 : 9,
-              xLarge: checker ? 12 : 9,
+              big: 5,
+              large: 9,
+              xLarge: 9,
```
Comment (afjlambert) [inline apps/client/src/components/Layouts/BaseLayout.tsx:37]: Oh wow, was the difference in columns actually not supposed to change?

Comment (svenjens) [reply apps/client/src/components/Layouts/BaseLayout.tsx:37]: It didn't change

**Review by svenjens — APPROVED** [review]

File: apps/client/src/components/StepByStepNavigation/StepByStepNavigation.tsx (hunk 1 above)
Comment (afjlambert) [reply apps/client/src/components/StepByStepNavigation/StepByStepNavigation.tsx]: What if you change the import to `import passPropsToChildren from "../../utils/usePassPropsToChildren";`?

File: apps/client/src/utils/usePassPropsToChildren.ts [hunk 3]
```diff
@@ -8,13 +8,13 @@ type Callback = (index?: number) => any;
  *
  * Use in case we need to pass props to children.
  * @param propsOrCallback this could be an object or a callback with the index as a parameter
- * @param childrenProps React children
+ * @param childrenProp React children
```
Comment (afjlambert) [inline apps/client/src/utils/usePassPropsToChildren.ts]: Can you change the order of these `@params` too?

**Review by afjlambert — APPROVED** [review]

File: apps/client/src/components/StepByStepNavigation/StepByStepNavigation.tsx (hunk 1 above)
Comment (robinpiets) [reply apps/client/src/components/StepByStepNavigation/StepByStepNavigation.tsx]: Good point! Works

Comment (robinpiets) [reply apps/client/src/components/StepByStepNavigation/StepByStepNavigation.tsx]: Fixed https://github.com/Amsterdam/vergunningcheck/pull/440/commits/bdd53468db1949504cda9c5fdc975bec0298e21e

File: apps/client/src/utils/usePassPropsToChildren.ts (hunk 3 above)
Comment (robinpiets) [reply apps/client/src/utils/usePassPropsToChildren.ts]: All fixed https://github.com/Amsterdam/vergunningcheck/pull/440/commits/bdd53468db1949504cda9c5fdc975bec0298e21e

**Pull request merged**